component/pyroscope/scrape/scrape_loop.go

```diff
@@ -228,7 +228,7 @@ func (t *scrapeLoop) scrape() {
 		}
 	}
 	if err := t.fetchProfile(scrapeCtx, profileType, buf); err != nil {
-		level.Debug(t.logger).Log("msg", "fetch profile failed", "target", t.Labels().String(), "err", err)
+		level.Error(t.logger).Log("msg", "fetch profile failed", "target", t.Labels().String(), "err", err)
```

Comment:
It makes sense to report this, but if the log line is expected to be printed a lot maybe this should be a metric? That way we can have an alert as well.

Reply:
lets try like this in dev and then revert and make it metric when we have them too much.

Also do you know if scraping metric produces error? It feels they should be aligned and have similar experience.